src/components/fee-summary/SbcFeeSummary.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <v-card>
+    <header class="font-weight-bold px-3 py-3">
+      <slot name="header">Fee Summary</slot>
+    </header>
+
+    <div v-show="fetchError">
+      <v-alert color="error" icon="warning" outlined>{{fetchError}}</v-alert>
+    </div>
+
+    <v-slide-y-transition group tag="ul" class="fee-list" v-show="!fetchError">
+      <template
+        v-for="lineItem in fees"
+        >
+        <li class="container fee-list__item"
+          :key="lineItem.filingType"
+          >
+          <div class="fee-list__item-name">{{filingLabel ? filingLabel : lineItem.filingType}}</div>
+          <div class="fee-list__item-value" v-if="lineItem.fee > 0">{{lineItem.fee | currency}}</div>
+          <div class="fee-list__item-value" v-else>No Fee</div>
+        </li>
+        <li class="container fee-list__item"
+          v-if="lineItem.priorityFees"
+          :key="lineItem.filingType+'-priority'"
+          >
+          <div class="fee-list__item-name pl-3">Priority Fee</div>
+          <div class="fee-list__item-value">{{lineItem.priorityFees | currency}}</div>
+        </li>
+        <li class="container fee-list__item"
+          v-if="lineItem.futureEffectiveFees"
+          :key="lineItem.filingType+'-futureEffective'"
+          >
+          <div class="fee-list__item-name pl-3">Future Effective Fee</div>
+          <div class="fee-list__item-value">{{lineItem.futureEffectiveFees | currency}}</div>
+        </li>
+        <li class="container fee-list__item"
+          v-if="lineItem.serviceFees"
+          :key="lineItem.filingType+'-transaction'"
+          >
+          <div class="fee-list__item-name pl-3">Service Fee</div>
+          <div class="fee-list__item-value">{{lineItem.serviceFees | currency}}</div>
+        </li>
+      </template>
+    </v-slide-y-transition>
+
+    <div class="container fee-total" v-show="!fetchError">
+      <div class="fee-total__name">Total Fees</div>
+      <div class="fee-total__currency">CAD</div>
+      <div class="fee-total__value">
+        <v-slide-y-reverse-transition name="slide" mode="out-in">
+          <div>{{totalFilingFees | currency}}</div>
+        </v-slide-y-reverse-transition>
+      </div>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop, Watch, Emit } from 'vue-property-decorator'
+import FeeServices from '@/services/fee.services'
+import { Fee } from '@/interfaces/fee-interface'
+import { FilingData } from '@/interfaces/FilingData'
```

Comment:
Can you import these from the shared packages?

Also update package.json for this component (need to include services).